### PR TITLE
[TACHYON-1292] Add a private constructor to the Factory class in TachyonMaster.java

### PR DESCRIPTION
**Pull request by susan101566**
https://tachyon.atlassian.net/browse/TACHYON-1292

### CODE REVIEW

**Comment by apc999** [comment]
Jenkins, test this please.

**Comment by gpang** [comment]
ok to test

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
I don't think your commit is associated with your github account. Could you follow the instructions here: https://help.github.com/articles/setting-your-email-in-git/

Thanks!

**Comment by aaudiber** [comment]
Jenkins, test this please.

File: servers/src/main/java/tachyon/master/TachyonMaster.java [hunk 1]
```diff
@@ -127,6 +127,8 @@ public static TachyonMaster createMaster() {
       }
       return new TachyonMaster();
     }
+
```
Comment (gpang) [inline servers/src/main/java/tachyon/master/TachyonMaster.java]: The findbugs analysis says there is trailing whitespace on this line.

**Comment by gpang** [comment]
Thanks for the improvement! I just left a minor style comment.

**Comment by yupeng9** [comment]
Jenkins, test this please

**Comment by aaudiber** [comment]
Moving to https://github.com/amplab/tachyon/pull/2246

**Pull request closed without merging**